plugins/resource_management/app/controllers/resource_management/domain_admin_controller.rb

```diff
@@ -226,9 +227,10 @@ def approve_package_request
         res.save!
       end
 
-      services.resource_management.apply_current_quota(@project_resources)
+      @services_with_error = services.resource_management.apply_current_quota(@project_resources)
       services.inquiry.set_inquiry_state(@inquiry.id, :approved, 'Approved')
-      render action: 'approve_request'
+      
+      render action: 'approved_request'
```

Comment:
The view having a different name than the controller is needlessly confusing, esp. for a single-letter difference.